src/cli/llm_client.rs

```diff
@@ -113,4 +128,62 @@ impl OpenAIClient {
 
         Ok(rec)
     }
+
+    async fn ask(&self, question: String, context: Vec<String>) -> Result<String, ShellError> {
+        let mut messages: Vec<ChatCompletionRequestMessage> = vec![];
+
+        messages.push(
+            ChatCompletionRequestSystemMessageArgs::default()
+                .content("You are a helpful assistant.")
```

Comment:
Going to need a comment on some context around why we need this :)

Reply:
Done